modules/job_status/app/controllers/job_status/job_statuses_controller.rb

```diff
@@ -0,0 +1,52 @@
+#-- encoding: UTF-8
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2020 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2017 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See docs/COPYRIGHT.rdoc for more details.
+#++
+
+module JobStatus
+  class JobStatusesController < ::ApplicationController
+    before_action :authorize_on_job
+    layout 'angular'
+
+    def show
+      # Frontend will handle rendering
+      # but we will need to render with layout
+      render html: '', layout: true
+    end
+
+    private
+
+    def authorize_on_job
+      return if JobStatus::Status
+        .where(job_id: params[:job_uuid], user_id: current_user.id)
+        .exists?
+
+      render_404
+    end
+  end
```

Comment:
Good idea. I did not see why my page requires a 'no_page' layout since the controller does not set any project. I removed it in favor of the common angular service and it appears to work as expected